server/controllers/automationStatsController.js

```diff
@@ -33,13 +33,25 @@ const queryCountResolver = queryCount => Number(queryCount[0].count);
  *
  * @returns {object} response reporting automation stats
  */
+
+const dateSubtractor = (id) => {
+  const date = id === 'days'
+    ? 0
```

Comment:
this check may be a bit dangerous, 'days' may be misspelt

Reply:
Thanks, @machariamarigi , however, `days` is an input id from the frontend. The user does not type it in so there is no possibility of it being misspelt.
As for the URL, the duration is now being passed as a query, not a parameter.

Reply:
Cool, however, consider how it's used in the frontend may change, it's better to avoid the bug altogether.

Reply:
I can't seem to figure out an alternative way to ensure that 1 is not assigned when one wants stats for a day. Any help will be greatly appreciated.

Reply:
I'd suggest raising an error when it's not "days", "months" etc. Will make it easier for any other developer working on this later.

Reply:
Thanks @machariamarigi , I have added a block of code to return an error when an invalid duration is received.